feat(cli): add flag to disable update notifications (#1740)
Redocly/redocly-cli: .changeset/little-badgers-occur.md

---
"@redocly/cli": patch
---

> allow to skip version updates using REDOCLY_UPDATE_NOTIFIER environment variable

Reviewer: ```suggestion
Added the `REDOCLY_UPDATE_NOTIFIER` environment variable so that users can skip version updates.
```
